Update inline images pattern

anthonydillon opened the pull request:
## Done
Updated the inline-images pattern to match the image sizes as closely as possible using the spacing variables. Deprecated the `__img` element as it doesn't always target the image. Created `__item` as a generic replacement.

## QA
- Pull code and run `gulp jekyll`
- Go to http://127.0.0.1:4000/vanilla-framework/examples/patterns/inline-images/
- Check that the logos are similar to the spec with the following caveat (padding: 48px as its the largest spacing we have)

## Details
Fixes https://github.com/vanilla-framework/vanilla-framework/issues/1039

## Screenshots
![10576587](https://cloud.githubusercontent.com/assets/1413534/25409766/3ce382bc-2a0b-11e7-83b3-fac0dbaf6855.png)

barrymcgee commented:
The sass-llinter is not happy.

<img width="747" alt="screen shot 2017-04-26 at 10 36 19" src="https://cloud.githubusercontent.com/assets/505570/25427994/389d1b62-2a6c-11e7-9c5d-5b4b6e555571.png">

anthonydillon commented:
Thanks @barrymcgee.
1. I have fixed the lint issues
2. Drive by fixed the spellchecker to check all files not just the homepage
3. Reverted the images in the inline images example

anthonydillon commented:
New image for design if needed:
![inline images - vanilla framework - examples](https://cloud.githubusercontent.com/assets/1413534/25430107/4a921190-2a73-11e7-8199-c268bc337973.png)

barrymcgee commented:
Thanks! :shipit:

anthonydillon commented:
Removed designed needed as they are caught on release.

anthonydillon commented:
@barrymcgee code ok?

barrymcgee commented:
@anthonydillon yep
